features/configuration/backtrace_exclusion_patterns.feature

```diff
@@ -1,7 +1,19 @@
 Feature: Excluding lines from the backtrace
 
-  To reduce the noise when diagnosing failures, RSpec excludes matching lines
-  from backtraces. The default exclusion patterns are:
+  To reduce the noise when diagnosing failures, RSpec can exclude lines
+  belonging to certain gems or matching given patterns.
+
+  If you want to filter out backtrace lines belonging to specific gems, you can
+  use `config.filter_gems_from_backtrace` like so:
+
+  ```
+  config.filter_gems_from_backtrace "ignored_gem", "another_ignored_gem",
+  ```
+
+  For more control over which lines to ignore, you can assign a list of
+  patterns the `backtrace_exclusion_patterns` option.
```

Comment:
Actually, should be "patterns to the", I think.  Besides assigning to `backtrace_exclusion_patterns`, you can also append to it, and that's probably worth calling out here.